### PR TITLE
Newsletters: add 323 (2024-10-04)

### PR DESCRIPTION
**Pull request by harding**
- [x] Lede, releases/RCs, topic entries @harding 
- [x] Merge summaries @Gustavojfe

### CODE REVIEW

File: _posts/en/newsletters/2024-10-04-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+sources].*
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BDK 1.0.0-beta.4][] is a release candidate of this library for
+  building wallets and other Bitcoin-enabled applications.  The original
```
Comment (jonatack) [inline _posts/en/newsletters/2024-10-04-newsletter.md:24]: rephrasing idea, feel free to ignore

```suggestion
  building wallets and other Bitcoin-enabled applications.  Changes in this release candidate include renaming the `bdk` Rust crate to `bdk_wallet` and extracting lower-layer modules to their own crates, like `bdk_chain`, ...
```

**Review by jonatack — COMMENTED** [review]
Looks good so far.

**Comment by Gustavojfe** [comment]
Just added all the merge summaries...

I switched the order of the Eclair PRs so that they follow the order in which they were merged, I think it reads better this way, but open to feedback. thanks

File: _posts/en/newsletters/2024-10-04-newsletter.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+*No significant news this week was found in any of our [sources][optech
+sources].*
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [BDK 1.0.0-beta.4][] is a release candidate of this library for
```
Comment (jirijakes) [inline _posts/en/newsletters/2024-10-04-newsletter.md:23]: [1.0.0-beta.5](https://github.com/bitcoindevkit/bdk/releases/tag/v1.0.0-beta.5) was published in the meantime.

**Comment by harding** [comment]
Added news item about btcd security announcement, added lede, reviewed and made small edits to @Gustavojfe section (thanks!), updated releases/RCs (including new Bitcoin Core release), and added topic entries.  Thanks everyone!

File: _posts/en/newsletters/2024-10-04-newsletter.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+
+- [BDK #1616][] signals opt-in [RBF][topic rbf] by default in the
+  `TxBuilder` class.  The caller can disable the signal by changing the
+  sequence number.
+
+- [BIPs #1600][] makes several changes to the [BIP85][] specification, including
+  specifying that `drng_reader.read` (responsible for reading random numbers) is
+  a first-class function rather than an evaluation. This update also clarifies
+  endianness handling, adds support for [testnet][topic testnet], includes a new
+  reference implementation in Python, clarifies that a [HD wallet][topic bip32]
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-10-04-newsletter.md:137]:
```suggestion
  reference implementation in Python, clarifies that an [HD wallet][topic bip32]
```

File: _posts/en/newsletters/2024-10-04-newsletter.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+  seed wallet import format (WIF) uses most significant bits, adds the
```
Comment (bitschmidty) [inline _posts/en/newsletters/2024-10-04-newsletter.md:138]:
```suggestion
  seed wallet import format (WIF) uses the most significant bits, adds the
```

**Review by bitschmidty — COMMENTED** [review]
Just a couple fixups

**Comment by bitschmidty** [comment]
Removed an unused reference link, squashed.

Thank you @harding and @Gustavojfe for authoring this week and @jirijakes @jonatack and @vostrnad for reviewing 🚀

**Pull request merged**